metadata.rb

```diff
@@ -7,3 +7,4 @@
 version          '4.0.2'
 depends          'apt', '>= 2.0'
 depends          'yum', '>= 3.0'
+depends          'lvm'
```

Comment:
needs version constraint `'>= 1.5.1'` since thats when the compile time attributes got released